src/index.html

```diff
@@ -4,20 +4,160 @@
+    <link href="/src/css/components/commerce-info.css" rel="stylesheet" />
+    <link href="/src/css/utils.css" rel="stylesheet" />
     <title>Document</title>
   </head>
   <body>
-    <header></header>
+    <header class="header container">
+      <nav class="menu-container">
+        <div class="header-mobile-container">
+          <img class="header-mobile-logo" src="/src/assets/home-logo.png" alt="company logo" />
+          <button class="menu-nav-bars">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor">
+              <path
+                fill-rule="evenodd"
+                d="M3 6.75A.75.75 0 0 1 3.75 6h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 6.75ZM3 12a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 12Zm0 5.25a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75a.75.75 0 0 1-.75-.75Z"
+                clip-rule="evenodd"
+              />
+            </svg>
+          </button>
+        </div>
+        <ul class="menu-links">
+          <li><img class="header-logo" src="/src/assets/home-logo.png" alt="company logo" /></li>
+          <li><a class="header-links" href="#home">ホーム</a></li>
+          <li><a class="header-links" href="#case-studies">制作事例</a></li>
+          <li><a class="header-links" href="#testimonials">お客様の声</a></li>
+          <li><a class="header-links" href="#faqs">よく頂く質問</a></li>
+        </ul>
+        <ul class="menu-contact">
+          <li><a class="contact-button" href="contact.html">お問い合わせ・資料請求</a></li>
+        </ul>
+      </nav>
+    </header>
     <main>
-      <section></section>
-      <section></section>
-      <section></section>
-      <section></section>
-      <section></section>
+      <section class="hero container">
+        <img src="/src/assets/1.jpg" alt="hero carousel" />
+      </section>
+      <div class="features-main-heading">
+        <h1 class="features-heading">今月の特集</h1>
+        <p class="features-subheading">This Month's Special</p>
+      </div>
+      <section class="feature container section">
+        <div class="features-img-content">
+          <img class="features-img" src="/src/assets/1.jpg" alt="" />
+        </div>
+        <div class="features-content">
+          <h2 class="features-title">ここにタイトルを入れます</h2>
+          <p class="features-description">
+            ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。
+          </p>
+        </div>
+      </section>
+      <section class="feature-list section">
+        <div class="feature-list-main-heading">
+          <h1 class="feature-list-heading">施工事例</h1>
+          <p class="feature-list-subheading">Example of Construction</p>
+        </div>
+        <div class="feature-list-card-container container">
+          <div class="feature-list-img-card">
+            <img class="card-img" src="/src/assets/1.jpg" alt="house 1" />
+            <h3 class="card-title">東京都XX様宅</h3>
```

Comment:
There is a rule of numbering the `heading` tag. 

For example, the `h1.feature-list-heading` exists above,  so following that, this tag is supposed to be `h2`.  Thus, the number of heading tags is sequentially.

Another example, 

```.html
<header>
  <h1 class="title">title</h1>
</header>
...
<section>
  <h2>Heading</h2>
  <dl>
    <dt>title of...</dt>
    <dd>detail of...</dd>
  </dl>
  <section>
    <h3>sub title</h3>
    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum</p>
    </section>
</section>
```

As an example, the `section` tag can be nesting.